### ISSUE TITLE
View transferred amounts on recurring transfers

### ISSUE THREAD

**Issue by ckpringle** [vegaprotocol/frontend-monorepo#4174]
## Story

As a user
I want to see how much was transferred as a result of recurring transfers (including those initiated by governance)
So that I understand how funds are moving on the network

## Acceptance Criteria
- [ ] In the case of a one off transfer, I can see the transferred amount and when it was transferred 
- [ ] In the case of a recurring transfer, I can see the latest transfer amount and when it was transferred 
- [ ] There is a way to see the full list of transfers resulting from a recurring transfer

## Tasks

- [ ] UX (if needed)
- [ ] Design (if needed)
- [ ] Team and stakeholder review
- [ ] Specs reviewed and created or adjusted
- [ ] Implementation
- [ ] Testing (unit and/or e2e)
- [ ] Code review
- [ ] QA review

## Additional details / background info

Core change to allow query for single trabsfer ID is now in scope: https://github.com/vegaprotocol/vega/issues/7461

I think this requires an additional change to Core to:
1. Be able to query the ledger entries API for a transfer ID 
2. Make sure the ledger entries API actually works given at the moment is it a bit broken as I understand it

Have ticketed this [here](https://github.com/vegaprotocol/vega/issues/9516)

**Comment by JonRay15** [comment]
Pretty sure this  blocked and is not going to be important enough to unblock. 

This would require:

1. The ability to retrieve a transfer based on its ID, so that we can find the recurring transfer set up by a proposal
2. The ability to then query the ledger entries associated to that transfer so you can see the actual movement of funds

1 is currently impossible due to Core limitation. If this is cleared we can explore 2.

**Comment by JonRay15** [comment]
This is pribably on the verge of "nice to have" ... but it would be useful ... however Phill needs to fix the ledger entries API. Follow up with him.
